devsetup/scripts/gen-ansibleee-ssh-key.sh

```diff
@@ -32,14 +32,13 @@ fi
 
 pushd ${OUTPUT_DIR}
 
-if oc get secret dataplane-ansible-ssh-private-key-secret -n ${NAMESPACE} 2>&1 1>/dev/null; then
-    echo "Secret dataplane-ansible-ssh-private-key-secret already exists."
-    echo "Delete it first to recreate:"
-    echo "oc delete secret dataplane-ansible-ssh-private-key-secret"
-    exit 0
-fi
-
 if [ ! -f ${SSH_KEY_FILE} ]; then
+    if oc get secret dataplane-ansible-ssh-private-key-secret -n ${NAMESPACE} 2>&1 1>/dev/null; then
+        echo "Secret dataplane-ansible-ssh-private-key-secret already exists."
+        echo "Delete it first to recreate:"
+        echo "oc delete secret dataplane-ansible-ssh-private-key-secret -n ${NAMESPACE}"
+        exit 0
+    fi
     ssh-keygen -f ${SSH_KEY_FILE} -N "" -t ${SSH_ALGORITHM} -b ${SSH_KEY_SIZE}
```

Comment:
Perhaps these conditions shouldn't be nested?

cond1: If no key file, then create it. 
cond2: If no secret give warning and fail.

Why make cond2 contingent on cond1? Don't we want cond2 to run regardless of outcome of cond1?

Reply:
if you already have the key file, the secret will be created or "updated" using the same key file, which isn't a problem.
But if you already have a secret and somehow you don't have a key file, it means the secret will get updated with a key that not match an existing node